backend/db/operations.py

```
@@ -0,0 +1,55 @@
+from typing import Any
+from backend.db.connection import get_database
+from backend.models.user import User, Review
+from bson import ObjectId
+
+# User operations -->
+async def create_user(user: User):
+    """Create a new user in the database."""
+    db = await get_database()
+    user_dict = user.dict()
+    result = await db.users.insert_one(user_dict)
```

Ruff

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)